Adding Fluss docs (#38801)
home-assistant/home-assistant.github.io: source/_integrations/fluss.markdown

---
title: Fluss+
description: Instructions on how to integrate Fluss+ devices within Home Assistant.
ha_category:
  - Button
ha_release: "2025.6"
ha_iot_class: Cloud Polling
ha_codeowners:
  - '@fluss'
ha_domain: fluss
ha_config_flow: true
ha_platforms:
  - button
ha_integration_type: integration
---

> The **Fluss+** {% term integration %} allows you to control [Fluss+](https://fluss.io/) devices through Home Assistant.

Reviewer: ```suggestion
The **Fluss+** {% term integration %} allows you to control [Fluss+](https://fluss.io/) devices through Home Assistant. Fluss allows you to use your phone as a key.
```
I think it would be good to mention what Fluss devices are used for. Feel free to edit my suggestion.

Author: I'll adjust. for more detail. thank you

## Prerequisites

- A Fluss+ account.
- The Fluss+ app installed on your smart phone.
- Fluss+ device connected to your Wi-Fi network and has internet access.
- During setup of the integration in Home Assistant, you will be prompted to input your API key.
   - You can request an API key in the Fluss+ app under your profile settings.

{% include integrations/config_flow.md %}

### Button

> 

Reviewer: I don't understand exactly what the button does when triggering the device via Wi-Fi. Does it turn the Fluss device on or off?

Author: In short terms, it sends a trigger command to the Fluss device and that command sends a trigger to a connected device (for example, the Fluss device would trigger a connected gate motor to open / close)

Essentially you would wire the Fluss device Physically to a device (for example, a gate/garage motor). In the Fluss app, you would then be able to connect the Fluss device to the Wi-Fi. Then you will have the ability to trigger the device with the button in the app.

For Home Assistant, you set up an api key which is linked to the Fluss devices you have access to. The home assistant button would then send a trigger command the specific Fluss device.

Reviewer: oh, I see! Thank you for explaining. You think it would be worth mentioning that in the docs?

Author: Sure thing, I've elaborate more in the docs.

> The buttons represent the amount of devices you have access to to trigger via Wi-Fi. E.g if you have 3 devices, you will see 3 buttons. 

Reviewer: ```suggestion
A button in Home Assistant represents a Fluss device you can trigger via Wi-Fi. For example, if you have 3 devices, you will see 3 buttons. 
```

---
title: Fluss+
description: Instructions on how to integrate Fluss+ devices within Home Assistant.
ha_category:
  - Button

> ha_release: "2024.11"

Reviewer: ```suggestion
ha_release: "2025.6"
```

ha_iot_class: Cloud Polling
ha_codeowners:
    -'@fluss'
ha_domain: fluss
ha_config_flow: true
ha_platforms:
  - button
ha_integration_type: integration

> related:
>   - docs: /docs/configuration/
>     title: Configuration file

Reviewer: ```suggestion
```
I don't think the config file needs to be edited, as there is a config flow.

---

The **Fluss+** {% term integration %} allows you to control [Fluss+](https://fluss.io/) devices through Home Assistant.

## Prerequisites

> - A Fluss+ account

Reviewer: ```suggestion
- A Fluss+ account.
```

- The Fluss+ app installed on your smart phone.
- Fluss+ device connected to your Wi-Fi network and has internet access.
- During setup of the integration in Home Assistant, you will be prompted to input your API key.
   - You can request an API key in the Fluss+ app under your profile settings.

{% include integrations/config_flow.md %}

> ## Setup
> To add Fluss+ to your installation, go to **Settings** -> **Devices & Services** in the UI, click the button with `+` sign and from the list of integrations select **Fluss+**.
> You will be prompted to enter your API key.
> Once the integration is set up, you will see a list of your Fluss+ devices.

Reviewer: ```suggestion
```
this is more or less a duplicate of what is provided by the include element above.

![image](https://github.com/user-attachments/assets/ee6f6ae1-0f62-4ea5-a4ce-40f0c4bb9563)

### Button

The buttons represent the amount of devices you have access to to trigger via Wi-Fi. E.g if you have 3 devices, you will see 3 buttons. 

Each button can be triggered via Wi-Fi as long it's connected to the internet. The button will be triggered when you press it in Home Assistant.

## Services

### Service `fluss.press`

Trigger a Fluss device by simulating a button press.

| Service data attribute | Optional | Description |
|-----------------------|----------|-------------|

> | `entity_id`           | No       | The entity ID of the Fluss button to press (e.g., `button.fluss_12345`). |

Reviewer: ```suggestion
| `entity_id`           | No       | The entity ID of the Fluss button to press (e.g., `button.fluss_12345`). |

## Removing the integration

This integration follows standard integration removal.

{% include integrations/remove_device_service.md %}
```
does the integration follow standard removal steps? or is there anything else I need to do? otherwise, you could use this section here
![image](https://github.com/user-attachments/assets/bc16f911-e147-4ed3-898c-ce85d68eb709)

ha_iot_class: Cloud Polling
ha_codeowners:
  - '@fluss'
ha_domain: fluss
ha_config_flow: true
ha_platforms:
  - button
ha_integration_type: integration
---

The **Fluss+** {% term integration %} allows you to control [Fluss+](https://fluss.io/) devices through Home Assistant. Fluss turns your phone into a remote key that can open your garage, your gate, your door or even the parking at your office - giving you complete control. The Home Assistant integration allows you to trigger Fluss+ devices via Wi-Fi to open or close a connected motor device.

## Prerequisites

- A Fluss+ device (Can order here: https://fluss.io/flussplus).
- A Fluss+ account.
- The Fluss+ app installed on your smart phone.
- A  Fluss+ device physcially connected to your device (for example, a garage door opener).
- Fluss+ device connected to your Wi-Fi network and has internet access.
- During setup of the integration in Home Assistant, you will be prompted to input your API key.
   - You can request an API key in the Fluss+ app under your profile settings.
   - The API key will youu to access your Fluss+ devices and trigger them through Home Assistant.

{% include integrations/config_flow.md %}

### Button

> A button in Home Assistant represents a Fluss device you have access to. You can prress the button forr a certain device via Wi-Fi and it will then send a command to the connected device (like a grage door opener).

Reviewer: ```suggestion
A button in Home Assistant represents a Fluss device you have access to. You can press the button for a certain device via Wi-Fi, and it will then send a command to the connected device (like a garage door opener).
```

#### Example

> If you have access to three devices in the Fluss+ app, named "Home Garage Door," "Front Gate," and "Office Park," Home Assistant will display three buttons, each labeled with the corresponding device name. Pressing the "Home Garage Door" button sends a command to the Fluss+ device connected to your garage door opener which will then trigger the motor to open / close. The same applies to the other devices.

Reviewer: ```suggestion

If you have access to three devices in the Fluss+ app, named "Home Garage Door," "Front Gate," and "Office Park," Home Assistant will display three buttons, each labeled with the corresponding device name. Pressing the "Home Garage Door" button sends a command to the Fluss+ device connected to your garage door opener, which will then trigger the motor to open / close. The same applies to the other devices.
```

---
title: Fluss+
description: Instructions on how to integrate Fluss+ devices within Home Assistant.
ha_category:
  - Button

> ha_release: "2025.6"

Reviewer: ```suggestion
ha_release: "2025.10"
```
to be checked when the integration is merged into core.

> ha_release: "2025.10"

Reviewer 2: ```suggestion
ha_release: "2026.1"
```

- During setup of the integration in Home Assistant, you will be prompted to input your API key.
   - You can request an API key in the Fluss+ app under your profile settings.
   - The API key will youu to access your Fluss+ devices and trigger them through Home Assistant.

{% include integrations/config_flow.md %}

### Button

A button in Home Assistant represents a Fluss device you have access to. You can press the button for a certain device via Wi-Fi, and it will then send a command to the connected device (like a garage door opener).

#### Example

If you have access to three devices in the Fluss+ app, named "Home Garage Door," "Front Gate," and "Office Park," Home Assistant will display three buttons, each labeled with the corresponding device name. Pressing the "Home Garage Door" button sends a command to the Fluss+ device connected to your garage door opener, which will then trigger the motor to open / close. The same applies to the other devices.

## Services

### Service `fluss.press`

Trigger a Fluss device by simulating a button press.

| Service data attribute | Optional | Description |
|-----------------------|----------|-------------|

> | `entity_id`           | No       | The entity ID of the Fluss button to press (e.g., `button.fluss_12345`). |

Reviewer: ```suggestion
| `entity_id`           | No       | The entity ID of the Fluss button to press (for example, `button.fluss_12345`). |
```
we [avoid use of e.g.](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/eg)

> | `entity_id`           | No       | The entity ID of the Fluss button to press (for example, `button.fluss_12345`). |

Reviewer 2: This is incorrect, this is just a button press and we don't need to explicitly mention how an entity platform service works

Reviewer: are you suggesting to remove the entire section?

Reviewer 2: Yes

Reviewer: OK, thank you for the catch. As it is release day, I will merge and create a follow-up PR to implement the comment.
